django/templatetags/tz.py

```diff
@@ -31,7 +31,7 @@ def utc(value):
     """
     Convert a datetime to UTC.
     """
-    return do_timezone(value, datetime.UTC)
+    return do_timezone(value, UTC)
```

Comment:
This is a fix for regression in efb7f9ced2dcf71294353596a265e3fd67faffeb, not related with this patch.

Reply:
It seems that we don't have tests for the `utc` filter :thinking: